migrations/versions/1030_buyer_email_domain_data_migration.py

```diff
@@ -0,0 +1,47 @@
+"""Migrate buyer email domains from .txt file to DB
+
+Revision ID: 1030
+Revises: 1020
+Create Date: 2017-10-13 11:18:22.683693
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1030'
+down_revision = '1020'
+
+buyer_domains_table = sa.Table(
+    'buyer_email_domains',
+    sa.MetaData(),
+    sa.Column('id', sa.Integer, primary_key=True),
+    sa.Column('domain_name', sa.String, nullable=True),
+)
+
+
+def upgrade():
+    try:
+        with open('./data/buyer-email-domains.txt') as f:
+            legacy_buyer_domains = [line.strip() for line in f]
+    except FileNotFoundError as e:
+        # When we eventually delete the file, we can exit the migration early
+        return
+
+    conn = op.get_bind()
+    for legacy_domain in legacy_buyer_domains:
+        # SELECT domain_name FROM buyer_email_domains WHERE domain_name == 'legacy-domain.org'
+        query = buyer_domains_table.select(
+            buyer_domains_table.c.domain_name == legacy_domain
+        ).with_only_columns([buyer_domains_table.c.domain_name])
+
+        result = conn.execute(query).first()
+        if not result:
+            # INSERT INTO buyer_email_domains (domain_name) VALUES ("legacy-domain.org");
+            query = buyer_domains_table.insert().values(domain_name=legacy_domain)
+            conn.execute(query)
+
+
+def downgrade():
+    pass
```

Comment:
Could do a thing to delete all domains that don't have a "create_buyer_domain" audit event associated with them, but... nah 😛

Reply:
These wouldn't have an audit event as we're creating them directly! Should they? Who would be the 'updated_by' user?

Reply:
Exactly - if they have no audit event then that means we could roll back this migration without deleting stuff that had been added subsequently (which does have an audit event).

But no, I don't think a rollback is really necessary.

And no, I don't think this migration should create audit events.

Reply:
Ahhh I misread your comment. Yes that makes sense, on all points 😸